Framework/Core/include/Framework/AnalysisDataModel.h

```diff
@@ -266,6 +266,14 @@ DECLARE_SOA_EXPRESSION_COLUMN(DetectorMap, detectorMap, uint8_t, //! Detector ma
                                 ifnode(aod::track::trdPattern > (uint8_t)0, static_cast<uint8_t>(o2::aod::track::TRD), (uint8_t)0x0) |
                                 ifnode((aod::track::tofChi2 >= 0.f) && (aod::track::tofExpMom > 0.f), static_cast<uint8_t>(o2::aod::track::TOF), (uint8_t)0x0));
 
+DECLARE_SOA_DYNAMIC_COLUMN(TOFExpTime, tofExpTime, //! Expected time for the track to reach the TOF
+                           [](float length, float tofExpMom, float mMassZSqared) -> float {
+                             if (tofExpMom <= 0.f) {
+                               return -999.f;
+                             }
+                             return length * sqrt((mMassZSqared) + (tofExpMom * tofExpMom)) / (0.0299792f * tofExpMom);
```

Comment:
```suggestion
                             return length * std::sqrt((mMassZSqared) + (tofExpMom * tofExpMom)) / (0.0299792f * tofExpMom);
```

Reply:
Can we use one of those constants? https://github.com/AliceO2Group/AliceO2/blob/dev/Common/Constants/include/CommonConstants/PhysicsConstants.h#L196 
If needed define the one you need: cm/ps?

Reply:
Indeed that was the best way to handle it, done!